k8s/eks/.env

```diff
@@ -0,0 +1,22 @@
+#### Please fill in the cluster details below: ####
+
+CLUSTER_NAME=default  # This value must be changed or the process will fail!
+REGION=eu-west-3 # Here we declare the AWS region where the cluster and our new EBS will be created
+CNI_COMBINATION=calico_weave # Or change to 'aws_vpc_cni_weave', depending on the desired CNI combination
+
+## NODEGROUP CONFIG ##
+# common
+AVAILABILITY_ZONE=eu-west-3a # For now it must be only one AZ because of the script logic
+EBS_SIZE=300 # As per original documentation, can be changed if needed
+# Infra
+AMI_INFRA="ami-030609f7e95c0f386" # Amazon EKS optimized Amazon Linux 2 v1.22 built on 08 Aug 2022.
+INSTANCE_TYPE_INFRA=c5.2xlarge  # 58 is the max number of pods for c5.2xlarge; if we put 110 for example, it will still cap it to 58 on the node
+DESIRED_CAPACIY_INFRA=2 
+VOLUME_SIZE_INFRA=80
+SSH_PATH_INFRA="~/.ssh/id_rsa.pub" # Please note that the key must exist, it will not be created automatically!
+# Plan
+AMI_PLAN="ami-030609f7e95c0f386" # Amazon EKS optimized Amazon Linux 2 v1.22 built on 08 Aug 2022.
```

Comment:
Could you please explain this? AMIs specified for the NGs are the same.

Reply:
Right now yes but there's nothing stopping me from putting 2 different IDs there since we have 2 different env keys for AMI IDs. If they have to match, having one would make more sense.

Reply:
Ah ok understood now. We can make this edit to remove one of the keys.

Reply:
Removed one variable so both nodegroups are using the same AMI.
Also, this variable has been removed from `.env` and we have switched to AMI per selected region.
Left an explanation on changing AMIs in the README.